codec_specs.md

```diff
@@ -152,7 +152,7 @@ A `Block` inside a `BlockGroup` **MUST** use `ReferenceBlock` elements if the fi
 A `Block` inside a `BlockGroup` **MUST** use `ReferenceBlock` elements if the `Block` doesn't contain a `Sequence Header OBU`.
 A `Block` with `frame_header_obu` where the `frame_type` is `INTRA_ONLY_FRAME` **MUST** use a `ReferenceBlock` with a value of 0 to reference itself.
 
-Initialization: The `CodecPrivate` consists of the `AV1CodecConfigurationRecord` described in section 2.3 of [@!AV1-ISOBMFF].
+Initialization: The `CodecPrivate` consists of the `AV1CodecConfigurationBox` described in section 2.3 of [@!AV1-ISOBMFF].
```

Comment:
What!? Nooooo, `Record` is correct! `Box` is wrong as `Box` contains the leading `avC1` bytes, which aren't used in Matroska.

Reply:
> What!? Nooooo, Record is correct! Box is wrong as Box contains the leading avC1 bytes, which aren't used in Matroska.

+1, [av1.md#codecprivate-1](https://github.com/ietf-wg-cellar/matroska-specification/blob/master/codec/av1.md#codecprivate-1) is wrong, https://aomediacodec.github.io/av1-isobmff/#av1codecconfigurationbox-syntax AV1CodecConfigurationBox (so Box) header is not in Matroska (not 4 bytes for size and no "avc1"string).

Reply:
Damn it, MP4 confuses me.